example.js

```diff
@@ -1,25 +1,28 @@
-'use strict';
-const fs = require('fs');
-const delay = require('delay');
-const aperture = require('.');
+import fs from 'node:fs';
+import timers from 'node:timers/promises';
+import {aperture, screens, audioDevices, videoCodecs} from './index.js';
```

Comment:
```suggestion
import {recorder, screens, audioDevices, videoCodecs} from './index.js';
```

Would make more sense to name it `recorder`, since we're doing changes anyway.

Reply:
What about exporting a `Recorder` class? Not sure about the demos example naming

```js
const recorder = new Recorder()
```

Reply:
You cannot record two things at once, so I don't think being able to create a new instance is very useful.

Reply:
I exported a `Recorder` instance named `recorder`.